### ISSUE TITLE
Shipping Containers Store less than vaults

### ISSUE THREAD

**Issue by Licflagg** [talrey/CreateDeco#163]
This is on the Fabric version, but Shipping containers store far less than vaults. 1 stack per block instead of 20. Is this an intended thing or a bug?

**Comment by Aiq0** [comment]
Yes, already reported a few times (for example #128), but not fixed yet

**Comment by talrey** [comment]
I've been fighting these things for over a year? Wow... well, they should finally be fixed for the next update, so thank you for your part in finding and reporting these issues.
